packages/components/src/styles/tailwind-variables.css

```diff
@@ -0,0 +1,11 @@
+
+/* Copied from output of `@tailwind utilities` */
+:root {
+  --tw-shadow: 0 0 #0000;
+  --tw-ring-inset: var(--tw-empty,/*!*/ /*!*/);
+  --tw-ring-offset-width: 0;
+  --tw-ring-offset-color: #fff;
+  --tw-ring-color: rgba(59, 130, 246, 0.5);
+  --tw-ring-offset-shadow: 0 0 #0000;
+  --tw-ring-shadow: 0 0 #0000;
+}
```

Comment:
still need to add a stylelint prettier integration to fix this automatically

Reply:
Why not include this in `global.css`?

Reply:
I could, it would just mean that we also have to copy/paste this for the app as well.